packages/babel-traverse/src/path/lib/virtual-types-validator.ts

```diff
@@ -26,66 +26,75 @@ import type * as t from "@babel/types";
 const { isCompatTag } = react;
 import type { VirtualTypeAliases } from "./virtual-types";
 
+type Opts<Object> = Partial<{
```

Comment:
Nit, can you call this `Obj` to avoid confusion with the global `Object` constructor?